pkgs/tools/networking/doggo/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  runCommand,
+  installShellFiles,
+}: let
+  buildDate = "2022-08-15T21:36:49";
+  buildCommit = "2cf9e7b";
+in
+  with lib;
+    buildGoModule rec {
+      pname = "doggo";
+      version = "0.5.4";
+      src = fetchFromGitHub {
+        owner = "mr-karan";
+        repo = pname;
+        rev = "v${version}";
+        sha256 = "sha256-6jNs8vigrwKk47Voe42J9QYMTP7KnNAtJ5vFZTUW680=";
+      };
+
+      vendorSha256 = "sha256-pyzu89HDFrMQqYJZC2vdqzOc6PiAbqhaTgYakmN0qj8=";
+      nativeBuildInputs = [installShellFiles];
+      subPackages = ["cmd/doggo"];
+
+      ldflags = [
+        "-w -s"
+        "-X main.buildVersion=${buildCommit}"
```

```suggestion
        "-X main.buildVersion=${version}"
```

Upstream [seems to use the version number here](https://github.com/mr-karan/doggo/blob/v0.5.4/.goreleaser.yml#L35) for tagged releases here, so we can probably get away with using `version` here, leaving out `buildCommit` and maybe even dropping build date entirely, since it is largely meaningless?